tests/framework/microvm.py

```diff
@@ -285,10 +285,9 @@ def kill(self):
         try:
             if self.firecracker_pid:
                 os.kill(self.firecracker_pid, signal.SIGKILL)
-        except ProcessLookupError:
-            LOG.exception("Process not found: %d", self.firecracker_pid)
-        except FileNotFoundError:
-            LOG.exception("PID file not found")
+        except:
+            LOG.error(self.log_data)
```

Comment:
Oh, so that's the difference between log.error and log.exception :laughing: 

Mh, we should be getting a backtrace anyway though, because we re-raise the exception